packages/schemas/src/foundations/jsonb-types/saml-application-configs.ts

```diff
@@ -0,0 +1,29 @@
+import { type ToZodObject } from '@logto/connector-kit';
+import { z } from 'zod';
+
+export type SamlAttributeMapping = Record<string, string>;
+
+export const samlAttributeMappingGuard = z.record(
+  z.string()
+) satisfies z.ZodType<SamlAttributeMapping>;
+
+export enum BindingType {
+  POST = 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST',
+  REDIRECT = 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect',
+}
+
+export type SamlSpMetadata = {
+  entityID: string;
+  acsURL: {
```

Comment:
The key name cases are slightly inconsistent. While `entityID` is fine as it's commonly used in SAML configurations, `acsURL` I'm not so sure. Should it be updated to `acsUrl` for consistency?